### PR TITLE
Fix GCC multi-line comment warning (-wcomment)

### PR DESCRIPTION
**Pull request by DhairyaBahl**
This should close the issue #1756 .

Thanks :)

### CODE REVIEW

**Comment by DhairyaBahl** [comment]
@ice0 @AYESDIE @Keyikedalube 
Kindly look into all my PRs and tell me if i have to improve any of them further.

Thanks for cooperation :)

File: synfig-studio/src/synfigapp/vectorizer/centerlineskeletonizer.cpp [hunk 1]
```diff
@@ -1385,17 +1386,18 @@ inline void Event::processSplitEvent() {
...
+ *               c     L     a'
+ *                \         /
+ *  m_generator =  b   x   b' = m_coGenerator
+ *                /         \
+ *               a     R     c'
 
-// Reflex vertices b and b' collide. Observe that a new reflex vertex may rise
+ * Reflex vertices b and b' collide. Observe that a new reflex vertex may rise
+*/
 // here.
```
[the remark this answers is not in the document]
Comment (DhairyaBahl) [reply synfig-studio/src/synfigapp/vectorizer/centerlineskeletonizer.cpp:1401]: Okay @AYESDIE correcting it. 

Thank you :)

**Comment by DhairyaBahl** [comment]
@AYESDIE Bro I updated it and i am trying to rebase this as well with github desktop but it shows up-to-date with origin/master and even with upstream/master ...What should I do ??

Thanks for support :)

**Comment by Keyikedalube** [comment]
@DhairyaBahl Build is not failing here.

Also, your commit is independent of someone else's new commit ie., they're not in conflict
So no need to rebase :)

**Comment by ice0** [comment]
Merged. Thank you!

**Comment by DhairyaBahl** [comment]
> @DhairyaBahl Build is not failing here.
> 
> Also, your commit is independent of someone else's new commit ie., they're not in conflict
> So no need to rebase :)

Okay I now finally understand what is rebasing and when to do it. Thanks :)

**Pull request merged**